### PR TITLE
Add a quick-start demo for the Dark Theme

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Does not work, I likely need a new incrementals version

### CODE REVIEW

File: demo/enable-theme.groovy [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// TODO: It should be in JCasC, but a demo patch is needed. We will use Groovy for now
```
Comment (timja) [inline demo/enable-theme.groovy:1]: what's wrong? I've used simple theme plugin from jcasc before

Comment (oleg-nenashev) [reply demo/enable-theme.groovy:1]: My demo is designed for a single YAML file, I need to modify it to the folder mode

**Comment by oleg-nenashev** [comment]
Hmmm. It still does not work though CSS seems to be OK. Am I missing something?

image

image

**Comment by oleg-nenashev** [comment]
Maybe I need to enable anonymous access to userContent...

**Comment by oleg-nenashev** [comment]
ready for review. I will implement the development node later, something is wrong with hosting the theme from `userContent`. Maybe the returned file type is now `plaintext` after the resent CSP improvements. CC @Wadeck @daniel-beck

**Comment by oleg-nenashev** [comment]
Will do

**Pull request merged**